way_to_home/static/src/components/routeForm/routeForm.js

```diff
@@ -20,9 +21,38 @@ class RouteSearchForm extends Component {
     }));
   }
 
-   getCurrentPosition = () => {
-     alert('get and write current position')
-   }
+   getCurrentPosition = (props) => {
+      if (!navigator.geolocation){
+        console.log("Geolocation is not supported by your browser")
```

Comment:
You probably need to show this message not in the console.

Reply:
where?

Reply:
where i must show error?

Reply:
For example, you can create the small alert message somewhere in the corner of the screen which will show ~5 seconds.

Reply:
or redirect to some page

Reply:
done